frontend/src/views/CellGuide/components/CellGuideCard/index.tsx

```diff
@@ -170,6 +209,8 @@ export default function CellGuideCard({
             <OntologyDagView
               key={cellTypeId}
```

Comment:
Can you make the key a combination of cellTypeId and tissueId so that changing either would reset the dag view.

Something like: 
```
key={`${cellTypeId}-${tissueId}`}
```

Reply:
@atarashansky - It seems a bit tricky. I think what we want is a re-render of the component correct?

If I try to update the key using the `selectedTissueId` getter from `useState`, I get the following error on tissue selection change from the dropdown

```
<OntologyDagView
              key={`${cellTypeId}-${selectedTissueId}`}
              cellTypeId={cellTypeId}
              tissueName={selectedTissue}
              tissueId={selectedTissueId}
              skinnyMode={skinnyMode}
            />
```

<img width="1147" alt="Screenshot 2023-08-23 at 1 27 57 PM" src="https://github.com/chanzuckerberg/single-cell-data-portal/assets/5581819/16733790-9ced-4e83-b89e-209da1707031">

Reply:
Hmm taking a look now!

Reply:
@prathapsridharan this seems to solve it!

https://github.com/chanzuckerberg/single-cell-data-portal/pull/5637

Feel free to merge 👍 Thanks!

Reply:
Thanks @timzchang